examples/sveltekit/package.json

```diff
@@ -13,12 +13,13 @@
 		"check:watch": "svelte-check --tsconfig ./tsconfig.json --watch",
 		"lint": "prettier --ignore-path .gitignore --check --plugin-search-dir=. . && eslint --ignore-path .gitignore .",
 		"format": "prettier --ignore-path .gitignore --write --plugin-search-dir=. .",
-		"postinstall": "node node_modules/puppeteer/install.mjs"
+		"postinstall": "playwright install --with-deps"
```

Comment:
Ideally we'd do this in the CI and leverage artifact caching, or, at least do `playwright install --with-deps chrome` (by default FireFox will also be installed). However, when passing a specific browser we get this error if it is already installed, which is a problem for local development. To keep things simple for now, I'm suggesting we just accept some slow down in CI.

```
 ╔═════════════════════════════════════════════════════════════════╗
│ ║ ATTENTION: "chrome" is already installed on the system!         ║
│ ║                                                                 ║
│ ║ "chrome" installation is not hermetic; installing newer version ║
│ ║ requires *removal* of a current installation first.             ║
│ ║                                                                 ║
│ ║ To *uninstall* current version and re-install latest "chrome":  ║
│ ║                                                                 ║
│ ║ - Close all running instances of "chrome", if any               ║
│ ║ - Use "--force" to install browser:                             ║
│ ║                                                                 ║
│ ║     pnpm exec playwright install --force chrome                 ║
│ ║                                                                 ║
│ ║ <3 Playwright Team                                              ║
│ ╚═════════════════════════════════════════════════════════════════╝
```